diff --git a/node_modules/rxdb/plugins/replication/index.cjs b/node_modules/rxdb/plugins/replication/index.cjs
index 2fb370b..3b66fc1 100644
--- a/node_modules/rxdb/plugins/replication/index.cjs
+++ b/node_modules/rxdb/plugins/replication/index.cjs
@@ -1,2 +1 @@
-const pkg = require('../../dist/cjs/plugins/replication/index.js');
-module.exports = pkg;
+export * from '../../dist/esm/plugins/replication/index.js';
diff --git a/node_modules/rxdb/plugins/storage-dexie/index.cjs b/node_modules/rxdb/plugins/storage-dexie/index.cjs
index 3d3d35b..a40e3da 100644
--- a/node_modules/rxdb/plugins/storage-dexie/index.cjs
+++ b/node_modules/rxdb/plugins/storage-dexie/index.cjs
@@ -1,2 +1 @@
-const pkg = require('../../dist/cjs/plugins/storage-dexie/index.js');
-module.exports = pkg;
+export * from '../../dist/esm/plugins/storage-dexie/index.js';
diff --git a/node_modules/rxdb/plugins/migration-schema/index.cjs b/node_modules/rxdb/plugins/migration-schema/index.cjs
index 8fc0e6a..8292c27 100644
--- a/node_modules/rxdb/plugins/migration-schema/index.cjs
+++ b/node_modules/rxdb/plugins/migration-schema/index.cjs
@@ -1,2 +1 @@
-const pkg = require('../../dist/cjs/plugins/migration-schema/index.js');
-module.exports = pkg;
+export * from '../../dist/esm/plugins/migration-schema/index.js';
